### ISSUE TITLE
Grid images are not shown as square

### ISSUE THREAD

**Issue by yanzm** [googlesamples/android-SliceViewer#3]
Checked version : 1.0.0-alpha1.1

I tried to set vector asset of home icon to GridRowBuilder.CellBuilder.addImage(). 
Then, the image was displayed in a horizontally long rectangle.
I expected square image like https://developer.android.com/guide/slices/templates#gridbuilder

<img width="184" alt="screen shot 2018-05-31 at 21 00 56" src="https://user-images.githubusercontent.com/455504/40781137-c2913f94-6515-11e8-9e18-b2f2cf7027b5.png">

```kotlin
    private fun createGridSlice(sliceUri: Uri): Slice? {
        return ListBuilder(context, sliceUri, ListBuilder.INFINITY)
                .addGridRow {
                    it.addCell {
                        it.addImage(
                                IconCompat.createWithResource(context, R.drawable.ic_home),
                                ListBuilder.LARGE_IMAGE)
                                .addTitleText("Title")
                                .addText("normal")
                    }
                    ...
                }
                .build()
    }
```

![screenshot_1527762296](https://user-images.githubusercontent.com/455504/40781046-7310dc5e-6515-11e8-94f2-7d9d263da8d8.png)

**Comment by yanzm** [comment]
I confirmed this problem is fixed with 1.0.0-alpha3. so I'll close this PR after new slice-viewer.apk is released.

**Comment by DougSig** [comment]
Really appreciate you reporting the issue! Closing since, like you said, the latest APK should fix it.
